lib/parser/provenance.cc

```diff
@@ -69,6 +69,18 @@ ProvenanceRange OffsetToProvenanceMappings::Map(std::size_t at) const {
   return provenanceMap_[low].range.Suffix(offset);
 }
 
+std::optional<std::size_t> OffsetToProvenanceMappings::ReverseMap(
+    Provenance at) const {
+  for (unsigned int i = 0; i < provenanceMap_.size(); i++) {
```

Comment:
`std::size_t i{0};`

Reply:
`++i` is preferred usage; it doesn't matter here, but with more general iterators it can save creation of a copy of the iterator.